lib/upgrade/6.2/upgrade.js

```diff
@@ -268,6 +276,12 @@ module.exports = function(options) {
 			.pipe(gulp.dest(DIR_SRC_CSS));
 	});
 
+	gulp.task('upgrade:version', function() {
+		var lfrThemeConfig = require('../../liferay_theme_config.js');
+
+		lfrThemeConfig.setVersion('1.0.0');
```

Comment:
How was `0.0.0` being set before? Can we not do it there?

Should `1.0.0` even be the value, as this is the beginning of development?

Reply:
Once a theme is created by the generator, the theme dev should have full control over the version property in the package.json. We shouldn't be modifying it.

Reply:
For reference: https://github.com/liferay/generator-liferay-theme/blob/master/generators/app/index.js#L316